### PR TITLE
feat: update template

### PR DESCRIPTION
**Pull request by leolangen**
- update version
- add cluster to config

### CODE REVIEW

File: src/Gateway.js [hunk 1]
```diff
@@ -35,14 +37,15 @@ function RequestGatewayToken() {
...
   console.log(`publicKey:${publicKey}`);
   return (
     /* gatekeeperSendsTransaction disabled is the default behavior and the flag can be omitted */
     <GatewayProvider
       wallet={wallet}
       gatekeeperNetwork={gatekeeperNetwork}
       stage={stage}
+      cluster={cluster}
       clusterUrl={clusterUrl}
       gatekeeperSendsTransaction={false}>
```
Comment (kevinhcolgan) [inline src/Gateway.js:50]: should remove as this is default

Comment (leolangen) [reply src/Gateway.js:50]: if you want to use a test network you have to set this to devnet, ortherwise you end up using devnet on the RC, but mainnet on the gatekeeper

Comment (kevinhcolgan) [reply src/Gateway.js:50]: I meant remove `gatekeeperSendsTransaction`

Comment (leolangen) [reply src/Gateway.js:50]: Oh right, sorry. In this case I added it for demonstration purposes with a comment above saying it's optional

**Review by kevinhcolgan — APPROVED** [review]

**Pull request merged**